github_pr.py

```diff
@@ -108,12 +111,70 @@ def github_list_prs(**args):
     elif 'head' in args and args['head']:
         prs = _load_prs_by_branch(**args)
         _print_prs(prs, **args)
+    elif 'filters' in args and args['filters']:
+        prs = github_filter_prs(**args)
+        _print_prs(prs, **args)
     else:
         prs = repo.get_pulls()
         _print_prs(prs, **args)
     return list_return_obj
 
 
+def github_filter_prs(**args):
+    """
+    Filters prs to return only what is contained in the filters
+    """
+    git_hub = Github(args['token'])
+    repo = git_hub.get_repo(args['repo'])
+    all_prs = [{'pr':repo.get_pull(pr.number), 'issue':repo.get_issue(pr.number)} for pr in repo.get_pulls()]
+    filters = {filter.split('=')[0]:filter.split('=')[1] for filter in args['filters'].split(',')}
```

Comment:
use `string.partition()` rather than splitting filter twice. Also, `filter()` is a builtin function in Python, which you are shadowing here.